src/basic/nulstr-util.c

```diff
@@ -3,15 +3,15 @@
 #include "nulstr-util.h"
 #include "string-util.h"
 
-bool nulstr_contains(const char *nulstr, const char *needle) {
+const char* nulstr_contains(const char *nulstr, const char *needle) {
```

Comment:
Hmm, then I prefer to rename this `nulstr_get()` or so, and introduce `nulstr_contains()` as
```
static inline bool nulstr_contains(const char *nulstr, const char *needle) {
        return nulstr_get(nulstr, needle);
}
```
C.f. `hashmap_contains()` vs `hashmap_get()`.

Reply:
That's a good suggestion indeed, thanks!